app.test.js

```diff
@@ -39,3 +38,41 @@ describe('/api/', () => {
     });
       })
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("Returns status 200 and a single article object", () => {
+    return request(app)
+    .get(`/api/articles/1`)
+    .expect(200)
+    .then(({body}) => {
+    const {article} = body;
+    // console.log(article)
+    expect(typeof article.title).toBe("string")
+    expect(typeof article.topic).toBe("string")
+    expect(typeof article.author).toBe("string")
+    expect(typeof article.body).toBe("string")
+    expect(typeof article.created_at).toBe("string")
+    expect(typeof article.votes).toBe("number")
+    expect(typeof article.article_img_url).toBe("string")
+    expect(typeof article.article_id).toBe("number")
+
```

Comment:
I'd delete lines 59 -69, it is not necessary to have both here 👍 remember to remove console logs 👀